notesystem/modes/convert_mode.py

```diff
@@ -392,12 +390,11 @@ def _convert_file(self, in_file: str, out_file: str) -> None:
             template = 'GitHub.html5'  # Default template (for html)
             template_str = f'--template {template}'
             if self._pandoc_options['template'] is not None:
-                if self._pandoc_options['template'] == '':
+                if self._pandoc_options['template'] == 'None':
```

Comment:
This changed how the `--pandoc-flag` is used. Do't forget to add this to the readme (usage/`--help`)